geps/gep-713.md

```diff
@@ -649,6 +1182,33 @@ const (
 )
 ```
 
+#### On targeted resources
```

Comment:
Given the size of this GEP and the provisional nature of this idea, maybe we can just link out to somewhere here instead of duplicating the content.

Reply:
I agree, but I'd prefer to get this merged, so that we have the text stored _somewhere_, and then move it around after. This PR does not finish the work to tidy up Policy Attachment, but it's become a bit unwieldy, so I'd like to concentrate on getting it merged and fixing afterwards.